fre/make/create_checkout_script.py

```diff
@@ -12,8 +12,30 @@
 fre_logger = logging.getLogger(__name__)
 
 def checkout_create(yamlfile, platform, target, no_parallel_checkout, jobs, execute, verbose):
-    """
-    Create the checkout script for bare-metal or container build
+    """Create the checkout script for bare-metal or container build
```

Comment:
Creates instead of create?

Reply:
Maybe checkout script needs a little explanation here?  For example, `checkout script that will clone the requisite repositories for model compilation`?